.tekton/tasks/tekton-results-upstream-e2e.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: tekton-results-upstream-e2e
+spec:
+  params:
+    - name: target_branch
+    - name: repo_url
+    - name: revision
+  workspaces:
+    - name: source
+    - name: kubeconfig-dir
+  steps:
+    - name: execute-tekton-results-upstream-e2e
+      image: quay.io/redhat-pipeline-service/ci-runner:$(params.target_branch)
+      workingDir: $(workspaces.source.path)
+      env:
+        - name: KUBECONFIG
+          value: "$(workspaces.kubeconfig-dir.path)/kubeconfig"
+      script: |
+        #! /usr/bin/env bash
+        cd $(workspaces.source.path)
+        git clone --no-checkout $(params.repo_url)
+        cd tektoncd-results
+        git config remote.origin.fetch '+refs/heads/downstream*:refs/remotes/origin/downstream*'
+        git fetch origin $(params.revision)
+        git checkout $(params.revision)
+        export SA_TOKEN_PATH=${SA_TOKEN_PATH:-"/tmp/tekton-results/tokens"}
+        export SSL_CERT_PATH=${SSL_CERT_PATH:="/tmp/tekton-results/ssl"}
+        mkdir -p ${SA_TOKEN_PATH} ${SSL_CERT_PATH}
+        kubectl get secrets tekton-results-tls -n tekton-results --template='{{index .data "tls.crt"}}' | base64 -d > ${SSL_CERT_PATH}/tekton-results-cert.pem
+        kubectl get secrets tekton-results-tls -n tekton-results --template='{{index .data "tls.key"}}' | base64 -d > ${SSL_CERT_PATH}/tekton-results-key.pem
+        oc annotate ingresses.config/cluster ingress.operator.openshift.io/default-enable-http2=true
```

Comment:
I think the right place to do this configuration is after this line..
https://github.com/openshift-pipelines/pipeline-service/blob/12862f674f40f65cf94d48691e3fe1a2d3bf95a6/ci/images/ci-runner/hack/bin/deploy-cluster.sh#L102

Reply:
Okay, should I cancel the current running, or will it run for now?